envs/conda_build_config.yaml

```diff
@@ -43,42 +42,29 @@ cython:
   - 0.29
 cython_blis:
   - 0.4.*
-dataclasses:
-  - 0.7
 dm_tree:
   - 0.1.*
 eigen:
   - 3.3.7
 ffmpeg:
-  - 4.2.0
-flatbuffers:
-  - 1.12.*
+  - 4.2
```

Comment:
I think we may want this as 4.2.0 still to fix the issue of avoiding the h264 stuff from anaconda..